xseed.sh

```diff
@@ -43,22 +43,65 @@ else
 fi
 
 # Use environment variables with descriptive default values
-TORRENT_CLIENT_NAME=${TORRENT_CLIENT_NAME:-Qbit}
-USENET_CLIENT_NAME=${USENET_CLIENT_NAME:-SABnzbd}
+# For multiple clients, use format: client1,client2,client3
+TORRENT_CLIENTS=${TORRENT_CLIENTS:-"Qbit"}
+USENET_CLIENTS=${USENET_CLIENTS:-"SABnzbd"}
```

Comment:
```suggestion
# format for up to as many clients as you need 
# Multiple clients: (${TORRENT_CLIENTS:-"Qbit","Qbit2"})
# Single Client: (${TORRENT_CLIENTS:-"Qbit"})
TORRENT_CLIENTS=(${TORRENT_CLIENTS:-"Qbit"})
USENET_CLIENTS=(${USENET_CLIENTS:-"SABnzbd"})
```

There's not need to parse as a string and convert it to an array and trim the whitespaces when you can just define it as an array, even if its only 1 client...